querqy-core/src/main/java/querqy/model/RewrittenQuery.java

```diff
@@ -0,0 +1,36 @@
+package querqy.model;
+
+import java.util.Collection;
+
+public class RewrittenQuery {
```

Comment:
We are introducing this class as a container for the rewriting output. This output will comprise more than the rewritten query. We should indicate this in the name, maybe `RewritingResult` or just `RewritingOutput`?

Reply:
I'm also not getting why we use delegation her (eg. `getUserQuery(): return expandedQuery.getUserQuery()`). Did I get the container concept wrong? I had assumed we'd add the loggingOutput etc. to this container later.

Reply:
It is a rewritten query plus metadata. All data in this object somehow describe the (rewritten) query. So I think RewrittenQuery should still be fine. The delegations are included simply to reduce the amount of refactoring regarding a massive amount of tests. 

If we change RewrittenQuery to RewritingOutput, the delegations should disappear. If we keep RewrittenQuery, we can keep them I think.

Reply:
To be honest, I'd like to see it as RewritingOutput (or whatever), without delegation, straight away as it would be a lot clearer. Haven't you already changed the tests (I see a lot of tests that just call `RewrittenQuery.getExpandedQuery()`)

Reply:
done